containers/cobol-gnucobol-2.2.0/main.cbl

```diff
@@ -0,0 +1,22 @@
+       identification division.
+       program-id. helloworld.
+       author. alvarodeleon.net.
+
+       environment division.
+       configuration section.
+       input-output section.
+
+       data division.
+
+       file section.
+
+       working-storage section.
+
+       77 mensaje picture x(32) value 'Hello World!!!'.
```

Comment:
Could you use `message` here and below instead?

Reply:
'message' is a reserved word I've switched it to 'imessage'